src/test/java/synapseawsconsolelogin/AuthTest.java

```diff
@@ -43,15 +43,15 @@
 	public void before() {
 		System.setProperty("TEAM_TO_ROLE_ARN_MAP","[{\"teamId\":\"123456\",\"roleArn\":\"arn:aws:iam::foo\"},{\"teamId\":\"345678\",\"roleArn\":\"arn:aws:iam::bar\"}]");
 		System.setProperty("AWS_REGION", "us-east-1");
-		System.setProperty("USER_CLAIMS", "userid,user_name");
+		System.setProperty("SESSION_TAG_CLAIMS", "userid,user_name");
 		System.setProperty(Auth.PROPERTIES_FILENAME_PARAMETER, "test.properties");
 	}
 	
 	@After
 	public void after() {
 		System.clearProperty("TEAM_TO_ROLE_ARN_MAP");
 		System.clearProperty("AWS_REGION");
-		System.clearProperty("USER_CLAIMS");
+		System.clearProperty("SESSION_TAG_CLAIMS");
```

Comment:
you replaced the `USER_CLAIMS` test with a test for `SESSION_TAG_CLAIMS`.  What about a test for the new  `SESSION_NAME_CLAIMS`?

Reply:
Good suggestion.  Will be included in the PR update.